### DISCUSSION TITLE
Welcome to CodingGuidelines Discussions!

### DISCUSSION THREAD

**Discussion by Keboo** [IntelliTect/CodingStandards#166] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by adamskt** [comment]
Because I can't attach a .md file:

# Architecture

## Solution Architecture

### Platform? Cloud/Mobile/Web/Data. How to pick which platform and when?

### Integration points.

### Tech stack

### Documentation

- Tooling?

### Suggested deliverables

### Problem statement

## Application Architecture 

### Project file structure

### Frameworks

## DevOps Architecture

### Roles - who is on the project and what are the responsibilities

- Who is testing the app?
- How are we testing the app?

### Documentation. What kinds of documentation and where are they stored?

- Local dev environment setup
- Running tests
- Contributing guidelines. Code reviews/Pull requests.
- Build/deployment process
- Configuration. Sources?

### Project process. Agile?
